### ISSUE TITLE
KLC Rule 6.4: Anchor placement clarification

### ISSUE THREAD

**Issue by Telemin** [KiCad/kicad-library#582]
Technically a footprint question but asking here as this repo contains KiCad_Library_Convention.txt

> 6.4.  For surface-mount devices, footprint anchor is placed in the middle with respect to device lead ends. (IPC-7351)

I want to clarify what this means.  "Lead ends" makes sense for most parts but can be ambiguous in the case of  parts that don't have symmetry in both X and Y dimensions, for example flex connectors designed to be soldered directly to board, or some (usually side entry) surface mount connectors. (See below for an example of flex lead designed to be soldered and secured via mounting holes)
In this case `check_kicad_mod.py` wants to put the component anchor in the middle of the footprint, but that doesn't feel like a fit with the wording of KLC 6.4,and my initial instinct was to place the anchor at the centre of the pad array.  I'm also not sure how much this actually matters, but it's something that has come up for a couple of footprints I have made recently and intend to submit soon.

tl;dr In cases like this does the anchor go at the centre of the entire footprint, or at the centre of the pad array?

![2016-06-17-003006_1920x1080_scrot](https://cloud.githubusercontent.com/assets/272538/16136486/d5a9bb0e-3422-11e6-9dfc-353d6762f141.png)

**Comment by SchrodingersGat** [comment]
See new KLC here - http://kicad-pcb.org/libraries/klc/F6.2/

Sorry that you never received an answer to this.
